processor/groupbytraceprocessor/config.go

```diff
@@ -25,7 +25,7 @@ type Config struct {
 	// Default: 1_000_000.
 	NumTraces int `mapstructure:"num_traces"`
 
-	// NumWorkers is a number of workers processing event queue. Should be equal to physical processors number.
+	// NumWorkers is a number of workers processing event queue.
```

Comment:
Maybe we could still advise a baseline of GOMAXPROCS not as a recommendation but as a starting point for optimization?

Reply:
I have updated the README.